Components/RegisterScreen.js

```diff
@@ -1,77 +1,112 @@
-import React from 'react';
+import React, {Component} from 'react';
 import { StyleSheet, TextInput, Text, View, Button, Image } from 'react-native';
 import { createStackNavigator } from 'react-navigation';
 import logo from './images/logo.png';
+import { connect } from 'react-redux';
+import {submitNewUserInformation} from './helperFunctions/Login';
+import {updateUserObject, updateIsUserLoggedIn} from '../actions';
 
-// let submitUserLoginInformation = (event, dispatch, history) => {
-//     event.preventDefault();
-//     let email = event.target.email.value;
-//     let password = event.target.password.value;
-//     console.log(email, password)
-//     event.target.reset();
-// };
+class RegisterScreenDumb extends Component {
+    constructor(props) {
+        super(props);
+        this.state = { email: '', password: '' };
+      }
 
-let RegisterScreen = ({props}) =>
-<View style={styles.container}>
+    render() {
+        let {navigation, dispatch} = this.props;
+        let {email, password} = this.state;
+
+        let registerUser = () => {
+            submitNewUserInformation(email, password)
+            .then(res => dispatch(updateUserObject(res)))
+            .then(dispatch(updateIsUserLoggedIn()))
+            .then(this.setState({email: '', password: ''}))
+            .then(navigation.navigate('Home'))
```

Comment:
why would you need to save email and password in state

Reply:
The local component is saving the state of the textinput components.  It resets it to an empty string once the submit button is pressed.

Reply:
lol, I read setState, but i didn't get to empty strings. I guess GraphQL weared me down